packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java

```diff
@@ -129,6 +132,33 @@ protected T prepareToRecycleView(@NonNull ThemedReactContext reactContext, T vie
     return view;
   }
 
+  @Override
+  public void onLayoutChange(View v,
```

Comment:
nit: It seems potentially error prone that this is only called when a transform is present. I.e. I could imagine someone adding code here, not realizing that it isn't always called.

Reply:
true, just didn't want to attach a listener if it was not needed. Should we write a comment maybe?

// Currently, the onLayout listener is only attached when transform-origin prop is being used.

Reply:
also removed the listener when transform-origin is null.